.vscode/launch.json

```diff
@@ -64,6 +65,27 @@
       "cwd": "${workspaceFolder}/cli",
       "args": ["uninstall", "--yes"],
       "buildFlags": "-tags=embed_manifests"
+    }, 
+    {
+      "name": "cli install",
+      "type": "go",
+      "request": "launch",
+      "mode": "debug",
+      "program": "${workspaceFolder}/cli",
+      "cwd": "${workspaceFolder}/cli",
+      "args": ["install", "--version", "<version_number>"], // Specify the desired version as the third argument in the command
```

Comment:
this is not valid `json`. Just making sure it was tested to work

Reply:
It looks like you can use env for that:
https://code.visualstudio.com/docs/editor/variables-reference#_environment-variables